propagators/opentelemetry-propagator-aws-xray/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@opentelemetry/propagator-aws-xray",
+  "version": "0.15.0",
+  "description": "OpenTelemetry AWS Xray propagator provides context propagation for systems that are using AWS X-Ray format.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "precompile": "tsc --version",
+    "compile": "tsc --build",
```

Comment:
in plugins `npm run version:update && tsc -p .` is used. Not sure what is correct here or if it is don't care anyway.

Reply:
I copied this from the `-ot-trace` module so figure sticking with it is ok

Reply:
Well generally the `version.ts` is used by instrumentations (because its required in the otel protocol). Since this package is not, its not needed in both the compile script or the `script.ts` file (so you can remove it)